.eslintrc.json

```diff
@@ -18,10 +18,7 @@
     "plugin:prettier/recommended",
     "plugin:unicorn/recommended",
     "plugin:cypress/recommended",
-    "prettier",
-    "prettier/react",
-    "prettier/unicorn",
-    "prettier/@typescript-eslint"
+    "prettier"
```

Comment:
If you want, you can remove this line too. https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21